openlibrary/macros/IABook.html

```diff
@@ -19,7 +19,7 @@ <h3 itemprop="name" class="booktitle">
            </h3>
         </div>
     </div>
-    $# The following (empty) <div> is to aid in formatting the layout.
```

Comment:
The benefit of `$#` is that this is visible to developers in the code, but it doesn't create a comment that needs to then be wastefully sent over the wire to patrons (extra kb's).

Reply:
Aha! I'll use the `$# detect-missing-i18n-skip-line` to handle these instead